src/primitives/transaction.h

```diff
@@ -286,73 +286,81 @@ struct CMutableTransaction;
  *   - CTxWitness wit;
  * - uint32_t nLockTime
  */
-template<typename Stream, typename Operation, typename TxType>
-inline void SerializeTransaction(TxType& tx, Stream& s, Operation ser_action) {
+template<typename Stream, typename TxType>
+inline void UnserializeTransaction(TxType& tx, Stream& s) {
```

Comment:
I agree, but whomever wrote serialize.h originally did not. The implicitly-defined method that classes get for deserialization is called `Unserialize`.